tests/testthat/test-tmt_qc_plots.R

```diff
@@ -0,0 +1,63 @@
+context("tmt_qc_plots")
+
+#### Setup ---------------------------------------------------------------------
+
+# Load some TMT PSM level data for tests
+load(system.file("testdata", "small_psm_tmt_phospho.rda",
+                 package = "camprotR"))
+load(system.file("testdata", "small_psm_tmt_total.rda",
+                 package = "camprotR"))
+
+#### Tests ---------------------------------------------------------------------
+# bit of a hacky test because can't use expect_equal_to_reference as the
+# object changes slightly every time you run the test.
```

Comment:
There's a package for regression testing for images but seems like overkill to me (https://github.com/r-lib/vdiffr). I'm happy with the 'hacky' test